packages/spatial/src/renderer/components/PostProcessingComponent.tsx

```diff
@@ -23,29 +23,75 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
 Ethereal Engine. All Rights Reserved.
 */
 
-import { useEffect } from 'react'
-
-import { defineComponent, useComponent, useEntityContext } from '@etherealengine/ecs'
-import { NO_PROXY_STEALTH } from '@etherealengine/hyperflux'
-
-import { defaultPostProcessingSchema, EffectPropsSchema } from '../effects/PostProcessing'
-import { configureEffectComposer } from '../functions/configureEffectComposer'
+import { Entity, defineComponent, useComponent, useEntityContext } from '@etherealengine/ecs'
+import { ErrorBoundary, NO_PROXY, getState, useHookstate } from '@etherealengine/hyperflux'
+import { Effect, EffectComposer, EffectPass, OutlineEffect, RenderPass } from 'postprocessing'
+import React, { Suspense, useEffect } from 'react'
+import { MotionBlurEffect, SSGIEffect, SSREffect, TRAAEffect } from 'realism-effects'
+import { ArrayCamera, Scene, WebGLRenderer } from 'three'
+import { CameraComponent } from '../../camera/components/CameraComponent'
+import { HighlightState } from '../HighlightState'
+import { RendererState } from '../RendererState'
+import { RendererComponent } from '../WebGLRendererSystem'
+import { ObjectLayers } from '../constants/ObjectLayers'
+import { PostProcessingEffectState } from '../effects/EffectRegistry'
+import { LinearTosRGBEffect } from '../effects/LinearTosRGBEffect'
 import { useScene } from './SceneComponents'
 
+declare module 'postprocessing' {
```

Comment:
this effect type override should be moved to each of the respective modules

Reply:
implemented